### ISSUE TITLE
Rule Change: (fill in)

### ISSUE THREAD

**Issue by IvanDLar** [eslint/eslint#16748]
### What rule do you want to change?

spaced-comment

### What change to do you want to make?

Implement suggestions

### How do you think the change should be implemented?

A new option

### Example code

```js
"spaced-comment": [
    "error", 
     "always",
      {
      "block": {
          "spaces": "never"
       }
 }]


//Hello ----> Error
// Hello ---> Correct

/*Hello*/ ----> Correct
/* Hello */ --> Correct
```

### What does the rule currently do for this code?

The rule checks both the inline and the block comments.

### What will the rule do after it's changed?

This rule will ignore the block comments, so that the rule is only enforced on the inline comment.

### Participation

- [X] I am willing to submit a pull request to implement this change.

**Comment by mdjermanovic** [comment]
Hi @IvanDLar, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we are not adding new options to the `spaced-comment` rule. You can always copy the existing rule into a custom rule and make it behave the way you want.
